torch/jit/_fuser.py

```diff
@@ -74,8 +74,11 @@ def _get_differentiable_graph_node(node, diff_node):
                 _get_differentiable_graph_node(n, diff_node)
 
 def _graph_for(self, *args, **kwargs):
+    return _script_method_graph_for(self, self, *args, **kwargs)
+
+def _script_method_graph_for(self, parent, *args, **kwargs):
```

Comment:
what is `parent`  for a `ScriptMethod`? Initially, I thought it would be a module reference, but I'm not sure one can call `get_debug_state()` on a Module?

Reply:
So it is the owner ScriptModule of a ScriptMethod, which does have `get_debug_state()`
https://github.com/pytorch/pytorch/blob/fb9a32b7b40b9136f169cb73437f2d273136cb37/torch/csrc/jit/python/script_init.cpp#L1041-L1049